src/wizard/wizard.scss

```diff
@@ -1,168 +1,137 @@
+          color: t(iui-color-foreground-accessory);
         }
-        font-weight: $iui-font-weight-semibold;
       }
+    }
 
-      > .iui-wizards-step-track-before {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
-        }
+    .iui-wizard-step-name {
+      @include themed {
+        color: t(iui-color-foreground-positive);
       }
     }
 
-    > .iui-wizards-step-completed {
-      &.iui-clickable {
-        > .iui-wizards-step-circle {
-          @include iui-focus(iui-focus-positive-box-shadow);
-          cursor: pointer;
-          @media (prefers-reduced-motion: no-preference) {
-            transition: background-color $iui-speed-fast ease-out;
-          }
-
-          &:hover {
-            @include themed {
-              color: t(iui-color-foreground-accessory);
-              border: 1px solid t(iui-color-background-positive);
-              background-color: t(iui-color-background-positive);
-            }
-          }
-        }
+    .iui-wizard-circle {
+      @include themed {
+        color: t(iui-color-foreground-positive);
+        border-color: t(iui-color-foreground-positive);
       }
+    }
 
-      > .iui-wizards-step-title {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-        }
+    .iui-wizard-track-content::before,
+    .iui-wizard-track-content::after {
+      @include themed {
+        background-color: t(iui-color-foreground-positive);
       }
+    }
+  }
 
-      > .iui-wizards-step-circle {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-          border: 1px solid t(iui-color-foreground-positive);
-        }
-      }
+  &:first-child .iui-wizard-track-content::before,
+  &:last-child .iui-wizard-track-content::after {
+    background-color: transparent;
+  }
+}
 
-      > .iui-wizards-step-track {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
-        }
-      }
+@mixin iui-wizard-track-content {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  &::before,
+  &::after {
+    content: '';
+    height: $iui-xxs;
+    flex: 1 2 auto;
+    @include themed {
+      background-color: t(iui-color-background-5);
+    }
+  }
+
+  .iui-wizard-circle {
+    align-items: center;
+    border-radius: 50%;
+    display: flex;
+    width: $iui-l;
+    height: $iui-l;
+    justify-content: center;
+    overflow-wrap: break-word;
+    padding: 0;
+    position: relative;
+    text-align: center;
+    user-select: none;
+    transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out, color $iui-speed-fast ease-out;
+    @include themed {
+      border: 1px solid t(iui-color-background-5);
+      background-color: t(iui-color-background-1);
+      color: t(iui-text-color-muted);
     }
   }
 }
+
+@mixin iui-wizard-step-name {
+  text-align: center;
```

Comment:
This can be removed too, I think

Reply:
I think this is needed, but only when the wizard gets really cramped for space horizontally.  Screenshot of how it looks when removed.
![Screen Shot 2021-08-11 at 10 30 18 AM](https://user-images.githubusercontent.com/849817/129048053-e90c93bf-6662-48c4-a904-24f0113d6402.png)